### PR TITLE
Latest updates

### PR DESCRIPTION
**Pull request by IgorGanapolsky**

### CODE REVIEW

File: demo/build.gradle [hunk 1]
```diff
@@ -21,5 +21,5 @@ android {
 
 dependencies {
     compile project(':library')
-    compile 'com.android.support:appcompat-v7:22.0.0'
+    compile 'com.android.support:appcompat-v7:22.+'
```
Comment (HeinrichReimer) [inline demo/build.gradle]: Don't use `+` in gradle dependencies.

File: library/build.gradle [hunk 2]
```diff
@@ -20,9 +20,9 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:support-v4:22.0.0'
-    compile 'com.android.support:appcompat-v7:22.0.0'
-    compile 'com.android.support:palette-v7:21.0.3'
+    compile 'com.android.support:support-v4:22.+'
```
Comment (HeinrichReimer) [inline library/build.gradle]: Don't use `+` in gradle dependencies.

File: library/build.gradle [hunk 3]
```diff
@@ -20,9 +20,9 @@ android {
...
+    compile 'com.android.support:appcompat-v7:22.+'
```
Comment (HeinrichReimer) [inline library/build.gradle]: Don't use `+` in gradle dependencies.

File: library/build.gradle [hunk 4]
```diff
@@ -20,9 +20,9 @@ android {
...
+    compile 'com.android.support:palette-v7:22.+'
```
Comment (HeinrichReimer) [inline library/build.gradle]: Don't use `+` in gradle dependencies.

File: library/build.gradle [hunk 5]
```diff
@@ -20,9 +20,9 @@ android {
...
 }
 
-apply from: 'https://raw.github.com/chrisbanes/gradle-mvn-push/master/gradle-mvn-push.gradle'
+//apply from: 'https://raw.github.com/chrisbanes/gradle-mvn-push/master/gradle-mvn-push.gradle'
```
Comment (HeinrichReimer) [inline library/build.gradle]: Remove `//`

**Comment by HeinrichReimer** [comment]
Thank you!
I commented on some lines that must be changed before merging.

File: demo/build.gradle (hunk 1 above)
Comment (HeinrichReimer) [inline demo/build.gradle]: Android Studio tells you to *"Avoid using + in version numbers [...]"*

**Comment by IgorGanapolsky** [comment]
Updated, let me know.

**Comment by IgorGanapolsky** [comment]
Thanks for the very nice example.  Have some @changetip $1 bitcoin on me.

**Comment by HeinrichReimer** [comment]
Haha, I'm a student. I can't give you money :smile:

**Comment by changetip** [comment]
Hi @heinrichreimer, @IgorGanapolsky sent you a Bitcoin tip worth 4,248 bits ($1.00), and I'm here to deliver it ➔ **[collect your tip](https://www.changetip.com/collect/737189)**.


**[Learn more about ChangeTip](https://www.changetip.com/tip-online/github)**

**Comment by IgorGanapolsky** [comment]
Its the other way around.

**Comment by HeinrichReimer** [comment]
Ah, now I understand...
Thank you! :smiley::smiley::smiley::smiley:

**Pull request merged**